pkg/daemon/state/state.go

```diff
@@ -73,6 +74,13 @@ func (s *State) SetCardRemovalTime(removalTime time.Time) {
 	s.cardRemovalTime = removalTime
 }
 
+func (s *State) SetCurrentlyLoadedSoftware(command string) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.currentlyLoadedSoftware = command
+	log.Debug().Msgf("current software launched set to: %s", command)
```

Comment:
Since a bug stuck me here 3 days i think a debug statement ready to go is possibly fine. I can remove this if we don't want it